DOCS-1945: Magento 1 Connect Settings to Connect Gateways (#1006)
MultiSafepay/docs: content/integrations/plugins/magento1/faq/ship-automatically-in-magento1.md

---
title : "Can I set orders to 'Shipped' automatically in Magento 1?"
meta_title: "Magento 1 plugin automatic shipping - MultiSafepay Docs"
meta_description: "The MultiSafepay Documentation Center presents all relevant information about our Plugins and API. You can also find support pages for payment methods, tools and general questions as well as the contact details of our Support and Integration Teams."
---

> Within our Magento 1 plugin, all complete orders will automatically be set to the status _Shipped_ . This is needed to capture the money from AfterPay, E-Invoicing, Klarna, Pay After Delivery and Santander Betaal per Maand.

Reviewer: ```suggestion
Within our Magento 1 plugin, all complete orders are automatically set to the status _Shipped_ . This is needed to collect the funds from AfterPay, E-Invoicing, Klarna, Pay After Delivery and Santander Betaal per Maand.
```
